src/main/java/io/botcrafting/botcraft/core/handler/MessageHandler.java

```diff
@@ -1,109 +1,18 @@
 package io.botcrafting.botcraft.core.handler;
 
-import static io.botcrafting.botcraft.configuration.constant.CommandConstant.COMMAND_EASTER_EGG_LIST;
-import static io.botcrafting.botcraft.configuration.constant.CommandConstant.COMMAND_HELP;
-import static io.botcrafting.botcraft.configuration.constant.CommandConstant.COMMAND_HELP_EXCEPTION;
-import static io.botcrafting.botcraft.configuration.constant.CommandConstant.COMMAND_SEARCH_BOOK;
-import static io.botcrafting.botcraft.configuration.constant.CommandConstant.EASTER_EGG_CTHULHU_AWAKE;
-import static io.botcrafting.botcraft.configuration.constant.CommandConstant.EASTER_EGG_HELLO_WORLD;
-import static io.botcrafting.botcraft.configuration.constant.MessageConstant.*;
-import static io.botcrafting.botcraft.configuration.constant.UrlConstant.BOTCRAFT_API_BASE_IMAGES_URL;
-
-import java.util.Optional;
-
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import io.botcrafting.botcraft.configuration.constant.MessageConstant;
-import io.botcrafting.botcraft.core.model.Book;
+import io.botcrafting.botcraft.core.handler.chain.MessageChain;
 import io.botcrafting.botcraft.core.model.Message;
-import io.botcrafting.botcraft.core.service.BookService;
-import io.botcrafting.botcraft.core.service.MessageSenderService;
 
 @Service
 public class MessageHandler {
```

Comment:
Looks like this class is not necessary anymore since it's just proxying to Chain, if makes sense we should remove it and use the chain implementation directly.